base/config/keycloak/import/ltt-realm.json

```
@@ -158,7 +158,7 @@
          "backchannel.logout.revoke.offline.tokens": "false",
          "frontchannel.logout.url": "${KEYCLOAK_SHL_CREATOR_BASE}/logout",
          "backchannel.logout.url": "",
-         "post.logout.redirect.uris": "${KEYCLOAK_CPRO_LANDING_PAGE}"
+         "post.logout.redirect.uris": "${KEYCLOAK_SHL_CREATOR_LOGOUT_REDIRECT_URL}"
```

Would be a little more clearer adding `POST` to distinguish from a URL to log the user out, and a URL to return to after being logged out.

We don't need a config var for both of those URLs, so please disregard this as a nitpick for now

```suggestion
         "post.logout.redirect.uris": "${KEYCLOAK_SHL_CREATOR_POST_LOGOUT_REDIRECT_URL}"
```